root/edit/details/EditPlace.js

```diff
@@ -0,0 +1,126 @@
+ */
+
+import React from 'react';
+
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import FullChangeDiff from '../../static/scripts/edit/components/edit/FullChangeDiff';
+import FullChangeDiffContent from '../../static/scripts/edit/components/edit/FullChangeDiffContent';
+import {formatCoordinates} from '../../utility/coordinates';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type PlaceEditT = {
+  ...EditT,
+  display_data: {
+    address: CompT<string>,
+    area: CompT<AreaT>,
+    begin_date: CompT<PartialDateT>,
+    comment: CompT<string>,
+    coordinates: CompT<CoordinatesT>,
+    end_date: CompT<PartialDateT>,
+    ended: CompT<boolean>,
+    name: CompT<string>,
+    place: PlaceT,
+    type: CompT<PlaceTypeT>,
+  },
+};
+
+const EditPlace = ({edit}: {edit: PlaceEditT}) => {
+  const display = edit.display_data;
+  const place = display.place;
+  let notDiffContent = true;
+  return (
+    <table className="details edit-place">
+      <tbody>
+        <tr>
+          <th>{addColonText(l('Place'))}</th>
+          <td><DescriptiveLink entity={place} /></td>
+        </tr>
+
+        {display.name ? (
+          <Diff
+            label={addColonText(l('Name'))}
+            newText={display.name.new}
+            oldText={display.name.old}
+            split="\s+"
+          />
+        ) : null}
+        {display.comment ? (
+          <Diff
+            label={addColonText(l('Disambiguation'))}
+            newText={display.comment.new}
+            oldText={display.comment.old}
+            split="\s+"
+          />
+        ) : null}
+        {display.type ? (
+          <FullChangeDiff
+            label={addColonText(l('Type'))}
+            newText={display.type.new.name}
+            oldText={display.type.old.name}
+          />
+        ) : null}
+        {display.address ? (
+          <Diff
+            label={addColonText(l('Address'))}
+            newText={display.address.new}
+            oldText={display.address.old}
+            split="\s+"
+          />
+        ) : null}
+        {display.area && (notDiffContent = display.area.new.gid === display.area.old.gid) ? (
+          <FullChangeDiffContent
+            notDiffContent
+            label={addColonText(l('Area'))}
+            newContent={display.area.new
+                ? <DescriptiveLink entity={display.area.new} />
+                : ''}
+            oldContent={display.area.old
+                ? <DescriptiveLink entity={display.area.old} />
+                : ''}
+          />
+        ) : null}
+        {display.coordinates ? (
+          <Diff
+            label={addColonText(l('Coordinates'))}
+            newText={formatCoordinates(display.coordinates.new)}
+            oldText={formatCoordinates(display.coordinates.old)}
```

Comment:
formatCoordinates doesn't work properly if coordinates are missing (for example, if the edit is *adding* coordinates).

Example:
![Screenshot from 2019-12-06 11-53-44](https://user-images.githubusercontent.com/1069224/70313934-2d393c80-181f-11ea-988d-73a0869d3ce4.png)

Reply:
Made temporary changes to fix this :)

Reply:
This needs to be unkludged now. :) See https://github.com/metabrainz/musicbrainz-server/pull/1229#discussion_r355174613